staging_service/config.py

```diff
@@ -85,7 +77,3 @@ def _get_path_value(section: SectionProxy, key: str) -> str:
     expects them that way. Consider changing it later.
     """
     return str(Path(_get_value(section, key)).absolute().resolve())
-
-
-class MissingAuthToken(Exception):
```

Comment:
Removed this since AUTH_TOKEN is now in the config, and should throw errors like any other config value.